StreetViewButtons.js

```diff
@@ -132,9 +144,22 @@ L.StreetView = L.Control.extend({
       last = b;
 
       var tmpl = b._template;
+	  var latitude1 = center.lat - 0.0002;
```

Comment:
This block is not used for mapillary (which sets `needUrl` to false) — please remove it, for it looks like a hack.

Reply:
sorry i dont understand which line(s) to remove. Can you do it yourself if you have rights? i'm not used to work on github

Reply:
is that ok now?

Reply:
do you have rights to edit my fork?

Reply:
if I remove (delete) l.144, 146, var tmpl is not defined anymore

Reply:
Done, I've added a few commits to clean up.